### PR TITLE
Search testing

### PR DESCRIPTION
**Pull request by stacyp2006**
### Participating Group Members:

- Stacy Potten

### Code Highlights:

- Adds testing for all existing DOM elements: buttons, headings, inputs
- Add testing for method fire on button click for `searchByLocation`

### Have Tests Been Added?

- [ ] No.
- [ ] Yes, but all tests are not passing.
- [x] Yes, and all are passing.

### Message/Questions for reviewer:
- Tests for sad paths and additional methods will be on a separate branch when that functionality is completed. 
- Any other tests I should add? 

### Issues:

- Addresses #105 
- Closes #

### Screenshots (if appropriate):

\*

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: src/components/FindPark.vue [hunk 1]
```diff
@@ -21,7 +22,7 @@
       v-model="searchTerm"
     />
     <router-link v-if="searchTerm" to="/results">
-      <button class="search-button" @click="search">
+      <button id="search" class="search-button" @click="search">
```
Comment (nicolegooden) [inline src/components/FindPark.vue:25]: Good idea to add an `id` so these buttons can be targeted in the test suite!

**Review by nicolegooden — APPROVED** [review]
Nice job adding such a thorough test suite for `FindPark`.  I appreciate your hard work and how you split up the unit tests to make them more modular.

**Review by bretm9 — APPROVED** [review]
Looks great! When I implement the API request into the search bar we might need to add some more tests here, but this covers everything that currently exists! Great work! Test coverage is really coming along.

**Review by npdarrington — APPROVED** [review]
Thank you for testing the component! Very nice to see it working and rendering as intended.

**Pull request merged**